staking/stakingprecompile.md

```diff
@@ -0,0 +1,95 @@
+
+ - `is_nominator(address collator)` checks whether the specified address is currently a staking nominator
+ - `is_candidate(address collator)` - checks whether the specified address is currently a collator candidate
+ - `min_nomination()` gets the minimum nomination amount
+ - `join_candidates(uint256 amount)` allows the account to join the set of collator candidates with a specified bond amount
+ - `leave_candidates()` - immediately removes the account from the candidate pool to prevent others from selecting it as a collator and triggers unbonding after BondDuration rounds have elapsed
+ - `go_offline()` - temporarily leave the set of collator candidates without unbonding
+ - `go_online()` - rejoin the set of collator candidates after previously calling go_offline()
+ - `candidate_bond_more(uint256 more)` - collator candidate increases bond by specified amount
+ - `candidate_bond_less(uint256 less)` - collator candidate decreases bond by specified amount
+ - `nominate(address collator, uint256 amount)` - if the caller is not a nominator, this function adds them to the set of nominators. If the caller is already a nominator, then it adjusts their nomination amount
+ - `leave_nominators()` - leave the set of nominators and revoke all ongoing nominations
+ - `revoke_nominations(address collator)` - revoke a specific nomination
+ - `nominator_bond_more(address collator, uint256 more)` - nominator increases bond to a collator by specified amount
+ - `nominator_bond_less(address collator, uint256 less` - nominator decreases bond to a collator by specified amount
+
+## Interacting with the Staking Precompile
+
+### Checking Prerequisites
+
+ - Have MetaMask installed and [connected to Moonbase Alpha](/getting-started/moonbase/metamask/)
+ - Have an account with over {{networks.moonbase.staking.min_nom_stake}} tokens. You can get this from [Mission Control](/getting-started/moonbase/faucet/)
+
+### Remix Set Up
+1. Get a copy of the [latest staking interface](https://github.com/PureStake/moonbeam/blob/master/runtime/precompiles/src/StakingInterface.sol)
+2. Copy and paste the file contents into a Remix file named StakingInterface.sol
+
+![Copying and Pasting the Staking Interface into Remix](/images/staking/stakinginterface1.png)
+
+### Compile the Contract
+1. Click on the Compile Tab, second from top
+2. Compile [Staking Interface.sol](https://github.com/PureStake/moonbeam/blob/master/runtime/precompiles/src/StakingInterface.sol)
+
+![Compiling StakingInteface.sol](/images/staking/compile4.png)
+
+### Access the Contract
+1. Click on the Deploy and Run Tab, directly below the Compile Tab in Remix
+2. Make sure "Injected Web3" is selected in the Environment drop down
+3. Make sure “ParachainStaking - StakingInterface.sol” is selected in the Contract dropdown. Since this is a precompiled contract on Moonbase Alpha there is no need to deploy, instead we are going to provide the address of the precompile in the “AtAddress” Field
```

Comment:
```suggestion
3. Make sure “ParachainStaking - StakingInterface.sol” is selected in the Contract dropdown. Since this is a precompiled contract on Moonbase Alpha there is no need to deploy, instead we are going to provide the address of the precompile in the “At Address” Field
```